package asunit4.support {

	public class ErrorInMethodTest {
		
		[Test]
		public function throw_ArgumentError():void {
			throw new ArgumentError('generated by ErrorInMethodTest');
		}
		
	}
}
